jenkins_python/scripts/PullRequestReport.py

```diff
@@ -193,6 +238,27 @@ def buildTestReport(templateEnv):
 if pylintSummary['comments']:
     message += '   * %s comments to review\n' % pylintSummary['comments']
 
+message += ' * Py27 check: %s\n' % statusMap[failedPy27]['readStatus']
+if py27Summary['added.message']:
+    message += 'Imports for Python3 compatability missing in new files. Please fix this:\n'
```

Comment:
The idea is to keep the details out of the message. I think its sufficient to just check failedPy27 and put a message in saying that it failed our python3 compatibility checks. And another message checking idioms saying there are suggested fixes for newer python3 idioms. So 0-2 lines in the GitHub message